process_linux.go

```diff
@@ -236,5 +236,14 @@ func (p *initProcess) createNetworkInterfaces() error {
 }
 
 func (p *initProcess) signal(s os.Signal) error {
-	return p.cmd.Process.Signal(s)
+	return maybeSyscall(p.cmd.Process, s)
+}
+
+func maybeSyscall(process *os.Process, s os.Signal) error {
```

Comment:
@crosbymichael @LK4D4 Where do we stand on this.  Is there some safe way that I could always just call syscall.Kill and never call process.Signal